README.md

```diff
@@ -115,6 +115,39 @@ Server
 In many rails projects when you run the server with `rails s` Rails sets RAILS_ENV to "development".  If you do that with this app, you will connect to the development database on AWS (if you have permission to decrypt the value).  Instead, run `RAILS_ENV=local rails s` to start the server and `RAILS_ENV=local rails c` to run the console.
 
 
+Multithreading
+========================
+Our multithreading functionality is done through ActiveJob, which employs DelayedJob on the backend.
+ActiveJob gets better after rails 5.2, so keep in mind that there is some functionality that is not perfect, until we can upgrade rails.
+
+Backstory:  Rails runs on a single thread.  Even when you schedule asynchronous operations, s.a. sending a request with HTTParty, that request is executed on the same thread as the rest of the application.  Rails simulates multithreading with creative use of scheduling.
```

Comment:
I'd rethink this paragraph. Most programming languages would run an HTTP request synchronously, unless explicitly told not to — the next line of code might rely on the result of that so it couldn't, by default, be run in a different thread. I think the point here is that we want this processing to happen asynchronously, in a separate process, not slowing down the main server process. I believe you _could_ actually use threads in a Rails app, but as the process will block until all threads are completed it wouldn't really do what we want.

Also, what does `s.a.` stand for? Not aware of that abbreviation!

Reply:
HttParty gem runs in same thread, asynchronous execution.  

Sorry, "s.a." stands for "such as".  It's in the same family of use as "ex." ("example") and "etc." ("etcetera").